src/engine/controls/bpmcontrol.cpp

```diff
@@ -179,7 +184,7 @@ void BpmControl::slotTranslateBeatsEarlier(double v) {
     mixxx::BeatsPointer pBeats = m_pBeats;
     if (v > 0 && pBeats &&
             (pBeats->getCapabilities() & mixxx::Beats::BEATSCAP_TRANSLATE)) {
-        const int translate_dist = getSampleOfTrack().rate * -.01;
+        const int translate_dist = getSampleOfTrack().rate * (-.01 * v);
```

Comment:
Why `* v` which is supposed to be 1.0 and not intended for scaling the step size?

Reply:
Because this connected to a ControlPushButton. 
Mybe we should rename the slot to clarify this:
slotTranslateBeatsEarlierButtonChanged()

Reply:
This way, the controlObject can be used for gross and fine adjustments. The default value is 1.0, so nothing changes. But, at least on my controller, I changed the value to some smaller value, so I can have more fine adjustments without adding a new control.